python/orca/example/learn/pytorch/super_resolution/super_resolution.py

```diff
@@ -299,24 +301,29 @@ def optim_creator(model, config):
         }
     )
 
-    if not exists(model_dir):
-        makedirs(model_dir)
+    if exists(model_dir):
+        shutil.rmtree(model_dir)
```

Comment:
Does this work for hdfs? Or we only save the model to driver in this example?

Reply:
> Does this work for hdfs? Or we only save the model to driver in this example?

I believe this example do not involve hdfs